packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/Bitcoin/Content/Empty/BitcoinWelcome/template.js

```diff
@@ -0,0 +1,115 @@
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  box-sizing: border-box;
+  @media(min-width: 1200px) {
+    flex-direction: row;
+  }
+`
+const Cell = styled.div`
+  width: 100%;
+  padding: 15px 0 15px 15px;
+  box-sizing: border-box;
+  @media(min-width: 768px) { ${props => props.small ? '30%' : '35%'}; }
+
+`
+const CloseArrow = styled(Icon)`
+  height: 30px;
+  width: 30px;
+  margin-top: 20px;
+`
+const BitcoinWelcome = props => {
+  const { displayed, handleClick, handleRequest } = props
+
+  return (
+    <Wrapper displayed={displayed}>
+      <BitcoinImage name='half-bitcoin' />
+      <Container>
+        <Row>
+          <Cell small>
+            <Text size='24px' weight={300} color='brand-primary' uppercase>
+              <FormattedMessage id='scenes.transaction.bitcoin.content.empty.bitcoinwelcome.welcome' defaultMessage='Welcome to Bitcoin' />
+            </Text>
+            <Link href='https://blockchain.info/wallet/bitcoin-faq' size='16px' target='_blank' weight={300} uppercase>
+              <FormattedMessage id='scenes.transaction.bitcoin.content.empty.bitcoinwelcome.learnmore' defaultMessage='Learn More' />
+              <Icon name='right-arrow' color='brand-secondary' />
+            </Link>
+          </Cell>
+          <Cell>
+            <Text weight={300} color='brand-primary' uppercase>
+              <FormattedMessage id='scenes.transaction.bitcoin.content.empty.bitcoinwelcome.what' defaultMessage='What is bitcoin?' />
+            </Text>
+            <Separator />
+            <TextGroup inline>
+              <Text size='12px' weight={300}>
+                <FormattedMessage id='scenes.transaction.bitcoin.content.empty.bitcoinwelcome.currency' defaultMessage='Bitcoin is a digital currency, and is used like other assets in exchange for goods and services.' />
+              </Text>
+              <Text size='12px' weight={300}>
+                <FormattedMessage id='scenes.transaction.bitcoin.content.empty.bitcoinwelcome.transact' defaultMessage='Unlike traditional currencies and assets, bitcoin is easily portable, divisible, and irreversible.' />
+              </Text>
+            </TextGroup>
+          </Cell>
+          <Cell>
+            <Text weight={300} color='brand-primary' uppercase>
+              <FormattedMessage id='scenes.transaction.bitcoin.content.empty.bitcoinwelcome.why' defaultMessage='Why should I use it?' />
+            </Text>
+            <Separator />
+            <TextGroup inline>
+              <Text size='12px' weight={300}>
+                <FormattedMessage id='scenes.transaction.bitcoin.content.empty.bitcoinwelcome.fees' defaultMessage='As a global currency you can send bitcoin to anyone, anywhere in the world without worrying about cross border remittance fees.' />
+              </Text>
+            </TextGroup>
+          </Cell>
+        </Row>
+        <Row>
+          {/* use link container to set up conditional to send to buy if available */}
```

Comment:
Can you replace this line with `{/* TODO: trade-status */}`. That way we can find and replace all of them with the logic at the same time?